app/models.py

```diff
@@ -222,8 +217,10 @@ class Contact(db.Model):
 
     @staticmethod
     def from_json(as_dict):
+        as_dict = normalize_key_case(as_dict)
+
         try:
-            return Contact(contact_for=as_dict.get('contactFor', None),
+            return Contact(contact_for=as_dict.get('contact_for', None),
```

Comment:
It is a step towards trying to standardize everything in the API to snake_case rather than camelCase (snake_case in order to match Postgres fields).

(This method actually accepts either because the dictionary keys get normalized to snake in the line above).

We currently have a mix of these cases spread around which makes it much more of a pain in the neck to reuse and standardize API code because we have to handle both versions of every key name.